doc/dev/decisionrecords/Codestyle.md

```diff
@@ -99,43 +99,44 @@ Working Directory: $ProjectFileDir$
 
 ### Other IDEs
 
-We do not have support for other IDEs at the moment. If you use another editor and make one please
+We do not have support for other IDEs at the moment. If you use another editor and make one, please
 feel free to share it.
 
 ### Sorting Class Members
 
-Some of the classes in OTP have a lot of fields and methods. Keeping members sorted reduce the merge
+Some of the classes in OTP have a lot of fields and methods. Keeping members sorted reduces merge
 conflicts. Adding fields and methods to the end of the list will cause merge conflicts more often
-than inserting methods and fields in an ordered list. Fields and methods can be sorted in "feature"
-sections or alphabetically, but stick to it and respect it when adding new methods and fields.
+than will inserting methods and fields in an ordered list. Fields and methods can be sorted in
```

Comment:
Parsing out the sentence, it becomes "[Adding fields and methods... will cause merge conflicts] more often than [inserting methods and fields... (_will cause merge conflicts_)]," so I added the "will" to make the implied verb phrase more clear.

But this was a very pedantic point on my part, and I'll revert it since it might add more confusion for readers (which is the opposite of this PR's intention!)